apps/design-land/src/app/quantity-field/quantity-field.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
```

Comment:
Can you move these design land files be moved to examples like the others?

Reply:
changed